src/main/webapp/index.jsp

```diff
@@ -38,7 +38,10 @@
       style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
 
       <h1>CodeU Chat App</h1>
-      <h2>Welcome!</h2>
+      <h2>Welcome to ~Team 31~'s chat app!</h2>
+      <p>Who is Team 31, you ask? We are an east coast-based team of CodeUers (plus
+          our PA Will, who is in Sunnyvale): Ellen Fu, Joyce Zhao, Pete Destil,
```

Comment:
"Sunnyvale, CA" only because you mentioned the east coast, so might as well highlight where Sunnyvale is (or even just say California)